docs/src/dev-docs/utils/data/readers.rst

```diff
@@ -1,5 +1,24 @@
+Readers
+#######
+
+Parsers for obtaining *system* and *target* information from disk. Currently,
+``metatrain`` support the following libraries for reading data
+
+.. list-table::
+  :header-rows: 1
+
+  * - Library
+    - Supported targets
+    - Default file format
+  * - ``ase``
+    - system, energy, forces, stress, virials
+    - ``.xyz``, ``.extxyz``
```

Comment:
do we restrict to only these format, or do we accept anything that ASE supports?

Reply:
We try to read everything if the user gives `reader: ase`. If the user specificies nothing we throw an error that we don't know the extension but he can try to set a reader explicit.

Reply:
We try to read everything if the user explicitly sets a reader in the options. If the user sets nothing we try to guess from the file extensions and throw an error that we don't know the extension but give a hint that one can set an explicit reader.